src/lvl_script_commands.c

```diff
@@ -294,9 +290,7 @@ const struct NamedCommand game_rule_desc[] = {
   {"AlliesShareVision",             29},
   {"AlliesShareDrop",               30},
   {"AlliesShareCta",                31},
-  {"BarrackMaxPartySize",           32},
```

Comment:
I don't get why this is removed. I also do not get why it still works,.... same for TrainingRoomMaxLevel below.